src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/HotReload/ProjectHotReloadSession.cs

```diff
@@ -272,7 +286,7 @@ public async ValueTask StopAsync(CancellationToken cancellationToken)
 
         public ValueTask<bool> SupportsRestartAsync(CancellationToken cancellationToken)
         {
-            return new ValueTask<bool>(_callback.SupportsRestart);
+            return new ValueTask<bool>(_callback is HotReloadState);
```

Comment:
Is this correct? Should it be:

```suggestion
            return new ValueTask<bool>(_callback is HotReloadState || _callback.SupportsRestart);
```

Reply:
Or, per below comment, can `HotReloadState.SupportsRestart` be changed to return `true`, then the change here can be reverted?

Reply:
Sure.